0.18-rc: "redeclaration of ‘ptr’ with no linkage" compile time error

```
[ 14%] Building C object CMakeFiles/mill.dir/stack.c.o
/home/constantine/Desktop/libmill/stack.c: In function ‘mill_allocstackmem’:
/home/constantine/Desktop/libmill/stack.c:100:11: error: redeclaration of ‘ptr’ with no linkage
     void *ptr = malloc(mill_get_stack_size());
           ^
/home/constantine/Desktop/libmill/stack.c:82:11: note: previous declaration of ‘ptr’ was here
     void *ptr;
           ^
make[2]: *** [CMakeFiles/mill.dir/stack.c.o] Error 1
make[1]: *** [CMakeFiles/mill.dir/all] Error 2
make: *** [all] Error 2
```

Makefile was generated from CMake.

--- a/stack.c
+++ b/stack.c
@@ -97,7 +97,7 @@ static void *mill_allocstackmem(void) {
         return NULL;
     }
 #else
-    void *ptr = malloc(mill_get_stack_size());
+    ptr = malloc(mill_get_stack_size());
     if(mill_slow(!ptr)) {
         errno = ENOMEM;
         return NULL;

